PYTHIA6/AliPythia6/AliGenPythiaPlus.cxx

```diff
@@ -1381,81 +1339,365 @@ void AliGenPythiaPlus::GetSubEventTime()
   return;
 }
 
+///
+/// \return true if particle in central barrel acceptance
+///
+/// \param eta        particle pseudorapidity,  etamin=-etamax
+///
+Bool_t AliGenPythiaPlus::IsInBarrel(Float_t eta) const
+{
+  if( eta < fTriggerEta  ) 
+  return kTRUE;
+  else 
+  return kFALSE;
+}
```

Comment:
It is a copy paste from AliGenPythia, I can change it here and there or only here.